sway-ir/src/printer.rs

```diff
@@ -476,14 +475,26 @@ fn instruction_to_doc<'a>(
                     md_namer.meta_as_string(context, span_md_idx, true),
                 )))
             }
-            Instruction::StateLoad { load_val, key } => Doc::text_line(format!(
-                "state_load ptr {}, key {}{}",
+            Instruction::StateLoadQuadWord { load_val, key } => Doc::text_line(format!(
+                "state_load_quad_word ptr {}, ptr {}{}",
                 namer.name(context, load_val),
```

Comment:
You've switched the `key` keyword to be another `ptr`, which I think might be confusing.  When reading the IR it isn't going to be obvious which argument is which.  Maybe `state_load_quad_word ptr {}, key ptr {}{}` is a compromise?

Reply:
I like having `ptr` there to indicate that this is indeed a pointer to a b256, so I now have `key ptr {}` as per your recommendation.